tests/test_workers/test_tasks/test_build.py

```diff
@@ -352,7 +352,9 @@ def test_skopeo_copy_fail_max_retries(mock_run_cmd):
 @mock.patch('iib.workers.tasks.build._create_and_push_manifest_list')
 @mock.patch('iib.workers.tasks.build.get_legacy_support_packages')
 @mock.patch('iib.workers.tasks.build.validate_legacy_params_and_config')
+@mock.patch('iib.workers.tasks.build.gate_bundles')
 def test_handle_add_request(
```

Comment:
Could add a test for this where the gating fails? This should be done by mocking `gate_bundles` to raise an exception.

Reply:
Done